Pokedex_ios/View/Controller/PokemonListViewController.swift

```diff
@@ -61,6 +64,19 @@ class PokemonListViewController: UIViewController {
             self.navigationController?.present(sheet, animated: true, completion: nil)
         }
     }
+    
+    func setSearchView() {
+        pokemonSearchTextField.addTarget(self, action: #selector(PokemonListViewController.textFieldDidChange(_:)), for: .editingChanged)
+        searchView.layer.cornerRadius = 10
+    }
+
+    @objc func textFieldDidChange(_ textField: UITextField) {
+        guard let text = textField.text else { return }
+        if text.isEmpty {
```

Comment:
this shouldn happen here.
Just sent whichever value the textfield has to the viewmodel searchText.
Then just call viewModel.applyFilters()
(See below comment first)

Reply:
done!